components/common/BtechDataTable.js

```diff
@@ -1,21 +1,88 @@
+'use client';
 import React, { useCallback } from 'react';
 import MUIDataTable from 'mui-datatables';
 import { createTheme, ThemeProvider } from '@mui/material/styles';
+import LinkedIn from '@mui/icons-material/LinkedIn';
+
+function DataTable({ data, initialRows }) {
```

Comment:
why is same code being repeated in this and mtech data table? create another component and name it DataTable and then add it to BtechDataTable and MtechDataTable, please don't repeat code.